spec/requests/splash_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Splashes', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
-  The same thing is here, you've not added any test in this file, so you should kindly add some as requested.

Reply:
- [ ] As requested by the previous reviewer kindly add some tests for this file as there is no test.